checkov/cdk/checks/typescript/LambdaInVPC.yaml

```diff
@@ -0,0 +1,51 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_117
+  name: Ensure that AWS Lambda function is configured inside a VPC
+  category: GENERAL_SECURITY
+  framework: cdk
+scope:
+  languages:
+    - typescript
+definition:
+  or:
+    - pattern: new $PACKAGE.aws_lambda.Function(<ANY>, <ANY>, { <ANY> })
+      conditions:
+        - not_pattern: |
+            new $PACKAGE.aws_lambda.Function(<ANY>, <ANY>, { <ANY>, vpc: <ANY>, <ANY> } )
+        - metavariable: $PACKAGE
+          comparison: $PACKAGE == "aws-cdk-lib"
+    - pattern: new $PACKAGE.aws_lambda.CfnFunction(<ANY>, <ANY>, { <ANY> })
+      conditions:
+        - not_pattern: |
+            new $PACKAGE.aws_lambda.CfnFunction(<ANY>, <ANY>, { <ANY>, vpcConfig: <ANY>, <ANY> } )
+        - metavariable: $PACKAGE
+          comparison: $PACKAGE == "aws-cdk-lib"
```

Comment:
does it catch aliases ?

Reply:
there is no aliases in the package name (at least by code)